### ISSUE TITLE
Any plans to update for Dawntrail?

### ISSUE THREAD

**Issue by Krojack** [Sevii77/ffxiv_materialui_accent#105]
Not looking for a time as I know these things take time to update. Just wondering if there are plans to update this for DT is all. If there is then I'll wait, if not then I know I can start looking for something else. That's all.

**Comment by Sevii77** [comment]
This project will be deprecated and no longer supported. I've made a new theme with customisation at its core which I'll maintain.

It can be found at https://github.com/Sevii77/frost_ui

**Comment by Krojack** [comment]
Interesting... I'll be waiting for some previews.

Also not sure what Aetherment is. Everything looks to be Linux based. Will the end result be usable via Penumbra or no?

**Comment by Sevii77** [comment]
I'm on linux which is why the build scripts are for linux. But it will be Penumbra based yes. This repo is both the ui mod and plugin in one, the new one is spit into 2.

**Comment by McRaeMan** [comment]
Will there be a plan for TexTools as well?

**Comment by ShichiIzanagi** [comment]
> Will there be a plan for TexTools as well?

Textools has not been supported since 6.4 - and Sevii has stated before there are no plans to support a textools version anymore and frankly TT should not be used as a mod loader.

**Comment by McRaeMan** [comment]
Oh.... Damn I have been hella out of touch then lol. Well appreciate the heads up!

Will be looking forward to the new loader and new UI!

**Comment by Krojack** [comment]
Yeah, TexTools literally modifies game files and is dangerous to use. Penumbra is clean and safe, and as someone that's used both, there is no way I would go back to TexTools. Penumbra is a game changer when I started using it.

**Comment by McRaeMan** [comment]
Well I will go ahead and get right on that then. TT was what I was introduced to with MateriaUI and the only thing I knew how to use. Thank you guys!

**Comment by Gasteno** [comment]
AWWWW man. I love MaterialUI. It's very clean. does frostui provide the same theme? Also not seeing instructions to install it will penumbra atm. Is that available? :(. Would you allow someone to take on the project and continue?

**Comment by ShichiIzanagi** [comment]
> AWWWW man. I love MaterialUI. It's very clean. does frostui provide the same theme? Also not seeing instructions to install it will penumbra atm. Is that available? :(. Would you allow someone to take on the project and continue?

FrostUI has a Material UI Preset which is the same as usual, its still there. Please check the discord for the announcement!

**Comment by Gasteno** [comment]
> > AWWWW man. I love MaterialUI. It's very clean. does frostui provide the same theme? Also not seeing instructions to install it will penumbra atm. Is that available? :(. Would you allow someone to take on the project and continue?
> 
> FrostUI has a Material UI Preset which is the same as usual, its still there. Please check the discord for the announcement!

will do. Thx for all that you do. I'll check it out.

**Comment by Tykku** [comment]
It has extremely reduced options compared to material ui

**Comment by ShichiIzanagi** [comment]
Where is Material UI?
Skotlex's Material UI is no longer maintained and was discontinued prior to the release of FFXIV patch 7.0. However, you can still find the Material UI (MUI) download at the following link: https://github.com/skotlex/ffxiv-material-ui. Please be aware that MUI is outdated (Patch 6.4), and certain UI elements will be missing. Install at your own risk, and do not request any help or updates for the Material UI mod in this server.

Can I make Frost UI resemble Material UI?
Yes and no. Frost UI is not intended to be a direct successor to Material UI; it is its own theme. However, it is influenced by Material UI and offers various options to replicate Material UI as closely as possible. There will be intentional differences in the UI elements. Frost UI is not MUI 2.0. If you miss MUI and want an updated version for patch 7.0, you will need to manually update the theme yourself, find someone else who has updated it, or hope that Skotlex returns to update it.

From the new FAQ. If you want your usual MUI you gotta get used to the new presets and/or customize it yourself. or beg Skotlex comes back. Its extremely reduced BECAUSE its still in a very early stage. Give it time.

**Comment by Rellawing** [comment]
Frost UI is absolutely worth installing! It's a worthy successor!

**Comment by RyudoGaming** [comment]
Deffo Give Frost UI ago! whilst there are a few things I would like added and I am sure they will be, it you can easily recreate what you had in Material UI and more.

**Comment by Tykku** [comment]
Honestly the square ticks when pressing an action key is the one thing bugging me about frost

**Comment by AlexisBliss** [comment]
were can i find said discord please
